build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export DATASOURCE=github
```

Comment:
Shouldn't we be sourcing the versions from rubygems?

Reply:
In this case I think we need to change [datasource](https://github.com/renovatebot/renovate/blob/master/lib/datasource/rubygems/releases.ts#L10) a little bit to make `rubygems.org` as the default registry url.

Reply:
Yes, we should. Default registries should be inserted by datasource and not the extract function.

Reply:
👌

Reply:
Is this ready for update now?

Reply:
Yes, works equivalently in my local setup